src/components/DocumentationTopic.vue

```diff
@@ -426,6 +429,15 @@ export default {
       || (downloadNotAvailableSummary && downloadNotAvailableSummary.length)
       || (primaryContentSections && primaryContentSections.length)
     ),
+    viewMoreLink: ({
+      interfaceLanguage,
+      objcPath,
+      swiftPath,
+      normalizePath,
+    }) => (
+      interfaceLanguage === Language.objectiveC.key.api
+        ? normalizePath(objcPath) : normalizePath(swiftPath)
```

Comment:
You could simplify this to `normalizePath(ternaryExpression)` to only call that function once since it is used in both conditions.